src/components/znkHeader/components/znkHeader/znkHeader.component.js

```diff
@@ -47,6 +50,11 @@
                         if (hasProVersion) {
                             vm.purchaseState = PurchaseStateEnum.PRO.enum;
                             vm.subscriptionStatus = '.PROFILE_STATUS_PRO';
+                            if (!isTeacherApp) {
+                                globalVariablesProm.then(globalVariables => {
+                                    vm.showReviewCreditBtn = Object.keys(globalVariables.manualReviewInitProCredits).includes(ENV.serviceId);
```

Comment:
Comment:
If we have the current service ID with an amount for initial review credits under the globalVariable.manualReviewInitProCredits, then it means the current app is an app that uses review credits, so we need to show this component.

Reply:
done